client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -385,7 +385,7 @@ public MarshallerRegistry getMarshallerRegistry() {
    @Override
    public void stop() {
       if (isStarted()) {
-         log.debugf("Stopping remote cache manager %x", System.identityHashCode(this));
+         log.debugf("Stopping remote cache manager %x", (Integer) System.identityHashCode(this));
```

Comment:
I don't understand the need for this cast. Without it it invokes log.debugf(String,int) which wraps the int in an Object[] thus causing boxing it in an Integer anyway.

Reply:
@tristantarrant maybe is not necessary, I just add that cast because I'm receiving the following exceptions:
`java.lang.NoSuchMethodError: org.infinispan.client.hotrod.logging.Log.debugf(Ljava/lang/String;I)V`
This exception is happening when I use `RemoteCache<String, String> cache = SERVER_TEST.hotrod().create();`

Reply:
Where do you get the exception @gustavolira ? What version of jboss-logging are you using there?

The newer versions of jboss-logging have `debugf` overloads for several combinations of primitive parameters:

https://github.com/jboss-logging/jboss-logging/blob/master/src/main/java/org/jboss/logging/BasicLogger.java#L658

Older versions of jboss-logging don't have those overloads, so you'll get a `NoSuchMethodError`. The solution is to upgrade jboss-logging to the same version used in Infinispan.

Reply:
Got it, thanks @danberindei 
@tristantarrant fixed